docs/modules/ROOT/pages/how-tos/installation.adoc

```diff
@@ -0,0 +1,16 @@
+= Installation of the Operator
+
+The Operator provides a Kustomize folder which allows for easy installation:
+
+[source,shell]
+--
+kubectl create namespace lieutenant
+
+# CRDs (global scope)
+kubectl apply -k github.com/projectsyn/lieutenant-operator/deploy/crds
```

Comment:
Is there a way in Antora to define a version number in a variable?
Then we could use something like

```suggestion
kubectl apply -k github.com/projectsyn/lieutenant-operator/deploy/crds?ref=${antora_version}
```

Reply:
That's a good idea, but at the moment we're not yet versioning the documentation and are always pointing to the master branch. This is something we certainly need to do in the future.